### PR TITLE
Dev themes

### PR DESCRIPTION
**Pull request by bhajneet**
Add night, philosophiae, darbar sahib, blue on white, and yellow on blue themes

### CODE REVIEW

File: app/frontend/src/components/Display.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .display {
     height: 100%;
-    background: var(--display-background-color) var(--display-background-image) no-repeat center;
-    background-size: 100% auto;
+    background: var(--display-background-color) var(--display-background-image);
+    background-size: var(--display-background-size);
```
Comment (Harjot1Singh) [inline app/frontend/src/components/Display.css]: Is there a common enough use-case to warrant the introduction of a new var?

File: app/frontend/themes/DarbarSahib.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+:root {
```
Comment (Harjot1Singh) [inline app/frontend/themes/DarbarSahib.css]: Rename file to `Darbar Sahib.css`, as name of theme will be detected from CSS filename

File: app/frontend/src/components/Display.css (hunk 1 above)
Comment (bhajneet) [reply app/frontend/src/components/Display.css]: Anytime you want the background to be tiled or filled

Comment (Harjot1Singh) [reply app/frontend/src/components/Display.css]: I’m also OK with size being a separate parameter as you’ve done now

**Pull request closed without merging**